en/docs/learn/outbound-provisioning-with-salesforce.md

```diff
@@ -516,6 +516,14 @@ management console, this user should also appear in Salesforce.
 3.  Enter the username in the form of an email and enter the password.
     
     !!! note
+        When adding the username, make sure to add a unique email address as the username which must be 
```

Comment:
Shall do the following change? Do we need the example about the test@gmail.com username? Seems like a repetition.

```
When adding the username, make sure to add an email address that is unique across all Salesforce organizations.
The user might not be provisioned to Salesforce if the username already exists even in another Salesforce organization.

For more information see the [Salesforce documentation](https://help.salesforce.com/s/articleView?id=000325728&type=1).
```